src/lib/java/de/teamlapen/lib/lib/util/UtilLib.java

```diff
@@ -721,4 +724,13 @@ public static AxisAlignedBB MBtoAABB(@Nonnull MutableBoundingBox bb) {
         HUNDRED_EIGHTY,
         TWO_HUNDRED_SEVENTY
     }
+
+    @Nullable
+    public static DyeColor getColorForItem(Item item) {
+        if (!Tags.Items.DYES.contains(item)) return null;
+        Optional<DyeColor> color = Arrays.stream(DyeColor.values()).filter(dye -> dye.getTag().contains(item)).findFirst();
+        if (color.isPresent()) return color.get();
```

Comment:
Why not stick to the optional here. In general I would say they are indeed more elegant and fit better to Forge style. But not a big difference anyway.